src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      action="#"
```

Comment:
The opening tag of the `form` element and its attributes should be formatted with each attribute starting on a new line, with a 2-space indentation relative to the tag.